sdks/python/apache_beam/coders/coders.py

```diff
@@ -309,11 +318,6 @@ class ToStringCoder(Coder):
   """A default string coder used if no sink coder is specified."""
 
   def encode(self, value):
-    try:               # Python 2
-      if isinstance(value, unicode):
-        return value.encode('utf-8')
```

Comment:
Should we do the same `unicode = str` thing here?  (In the new version, we will raise an error if the value is a non-ascii unicode string; eg: `str(u'😋')` raises an error, while this worked before.)

Reply:
The `unicode = str` trick won't work here on Python 3:
```
>>> str('test'.encode('utf-8'))
"b'test'"
```
I've reverted this change to the previous solution